specification/apicenter/ApiCenter.Management/models.tsp

```diff
@@ -557,3 +589,88 @@ model DeletedServiceProperties {
   @doc("UTC date and time when the service was soft-deleted. The date conforms to the following format: yyyy-MM-ddTHH:mm:ssZ as specified by the ISO 8601 standard.")
   softDeletionDate?: utcDateTime;
 }
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "Existing service"
+@added(Versions.v2024_06_01_preview)
+@doc("API source properties.")
+model ApiSourceProperties {
+  @doc("Indicates if the specification should be imported along with metadata.")
+  shouldImportSpec?: ShouldImportSpec = ShouldImportSpec.OnDemand;
```

Comment:
>shouldImportSpec [](http://example.com/codeflow?start=2&length=16)

The name sounds like a boolean, but the values are enum. 
Can the name be changed to - ImportSpecCondition or something like that ? removing the 'should' ?

Reply:
Renamed to "importSpecification".